api/src/org/apache/cloudstack/api/command/admin/host/UpdateHostCmd.java

```diff
@@ -62,6 +61,9 @@
     @Parameter(name = ApiConstants.URL, type = CommandType.STRING, description = "the new uri for the secondary storage: nfs://host/path")
     private String url;
 
+    @Parameter(name = ApiConstants.ANNOTATION, type = CommandType.STRING, description = "Add an annotation to this host")
```

Comment:
Add version field in this argument to state from which version this change was introduced.